Done - 32 - user -dashboard-mockup-page

jlo2224 opened the pull request:
[https://github.com/gameboiss/ask-me/blob/main/.github/pull_request_template.md]: #

# PR Description

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

## Additional Context or Relevant Issues

- Separated user dashboard and company dashboard
- Removed 'pseudo' column from company listing
- Made user dashboard
- Made Dashboard route go to company or user route, depending on which type they are
- Modified testcafe test, as requested
- Fixed dashboard to work when user is logged in, as requested
- Removed all instances of '/home' and replaced with '/dashboard', as requested 
- Refactored '_dashboard.scss' to '_company-dashboard.scss' so that we could also have '_student-dashboard.scss'
- Re-styled student dashboard to look a bit nicer ****** This might need more changes
- While working on this, I got distracted and ended up changing all instances of 'Companys' with 'Companies' in the database

Issues: Only issue is figuring out which to direct to for admins

### [Issue Link](https://github.com/uhmanoalink/manoa-link/issues/32)

winstonco reviewed (changes requested):
As you can see, the test for the dashboard doesn't work. Here are some potential fixes:
1. Change the dashboard test to navigate by clicking the link
    * Instead of navigating to http://localhost:3000/dashboard, click on the link to it in the navbar
2. **[Preferred]** Fix Dashboard page, which is returning NotAuthorized even when the user is logged in
3. **[Also Preferred]** Change the SignIn component to redirect to /dashboard instead of /home
    * There should be no /home route. The default route should be /dashboard

jlo2224 commented:
- Modified testcafe test, as requested
- Fixed dashboard to work when user is logged in, as requested
- Removed all instances of '/home' and replaced with '/dashboard', as requested 
- Refactored '_dashboard.scss' to '_company-dashboard.scss' so that we could also have '_student-dashboard.scss'
- Re-styled student dashboard to look a bit nicer ****** This might need more changes
- While working on this, I got distracted and ended up changing all instances of 'Companys' with 'Companies' in the database
